deb-package-install.adoc: Add installation notes

leon-anavi opened the pull request:
Extend the installation guide with examples.

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

tkfu commented on docs/deb-package-install.adoc:
```diff
@@ -1,6 +1,10 @@
+Aktualizr makes .deb packages available via the https://github.com/advancedtelematic/aktualizr/releases[GitHub releases page]. Download and install the .deb package, for example using dpkg:
+
+----
+sudo dpkg -i aktualizr.deb
```
Might be better to recommend using apt instead; that way the dependencies will get installed automatically:

```
wget https://github.com/advancedtelematic/aktualizr/releases/download/2018.3/aktualizr.deb
sudo apt install ./aktualizr.deb
```

leon-anavi commented:
@tkfu, thank you for the prompt feedback. Following your recommendations I have updated the GitHub PR. 

The **wget** command includes a specific URL which most probably will vary depending on the release. Therefore, to avoid updates of the docs with each new release of Aktualizr, I modified the text and left only the installation command. Do you think that this way the doc is comprehensive and user friendly?  

Best regards, Leon
